ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/db/entity/ProcessVersionEntityData.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.restmodel.db.entity
```

Comment:
We should rethink what should be in this package. db.entity in REST model doesn't look too good...
Maybe we need another layer?